state-synchronizer/src/coordinator.rs

```diff
@@ -96,8 +97,10 @@ impl PendingLedgerInfos {
     /// Adds `new_li` to the queue of pending LI's
     fn add_li(&mut self, new_li: LedgerInfoWithSignatures) {
         if self.pending_li_queue.len() >= self.max_pending_li_limit {
-            debug!(
-                "[state sync] pending LI store reached max capacity, failed to add LI {}",
+            warn!(
+                LogSchema::new(LogEntry::ProcessChunkResponse),
+                "pending LI store reached max capacity {}, failed to add LI {}",
+                self.max_pending_li_limit,
```

Comment:
what does this mean? is this warning covered in your runbook? it isn't clear to me why this would happen or what if anything we as an operator can do ... should we expect to be patient or...?